test.js

```diff
@@ -827,3 +828,28 @@ test('Prints any other `--before-build` command error', (assert) => new Promise(
     'exits with whatever code the `--before-build` command returned'
   );
 }));
+
+test('debounce debounces', (assert) => new Promise((resolve) => {
+  assert.plan(1);
+
+  let timesRan = 0;
+  const countTimesRan = () => { timesRan += 1; };
+
+  const wait = 1;
+  const debouncedcountTimesRan = debounce(countTimesRan, wait);
+
+  debouncedcountTimesRan();
+  debouncedcountTimesRan();
+  debouncedcountTimesRan();
+  debouncedcountTimesRan();
+
+  const checkTimesRan = () => {
+    assert.is(timesRan, 1,
+      `only calls the passed function once when the returned function
+is called several times within the wait interval`
+    );
+    resolve();
+  };
+
+  setTimeout(checkTimesRan, wait + 1);
```

Comment:
I'd be surprised if incrementing an integer 4 times took even 5ms on anything resembling modern hardware, but better safe than sorry I suppose.